data/vibe/data/serialization.d

```diff
@@ -571,6 +579,67 @@ private template serializeValueImpl(Serializer, alias Policy) {
 	}
 }
 
+///
+package template isSerializerSupportSinkType (SerT, ObjT)
+{
+	enum isSerializerSupportSinkType = is(typeof(SerT.serializeSinkType!(ObjT)(ObjT.init)) == void);
+}
+
+///
+package template isObjectSupportSinkType (ObjectT)
```

Comment:
```suggestion
package template isSinkSerializable (ObjectT)
```

I think the name is shorter and still fits with the existing style, what do you think ?